src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        int valueBallsToCreate = 3;
```

Comment:
this is not a constant
`BALLS_TO_CREATE` should be `private static final int BALLS_TO_CREATE = 3;` at the class level

Reply:
Im leave not constant because i dont know maybe in future it can be different value of balls. Okay)

Reply:
First time didnt see all your messages. In process)

Reply:
Also in video Mate sayed what constant dont use its very bad)))

Reply:
Bad i will put, no problem